care/facility/api/serializers/patient_consultation.py

```diff
@@ -555,7 +555,9 @@ def validate_create_symptoms(self, value):
             if item in counter:
                 # Reject if duplicate symptoms are provided
                 raise ValidationError("Duplicate symptoms are not allowed")
-            counter.add(item)
+            if not obj.get("cure_date"):
+                # skip duplicate symptom check for ones that has cure date
```

Comment:
This is not really skipping the duplicate symptom check right?

Reply:
yes it is, for ones with cure date specified.